### DISCUSSION TITLE
How to display the .openPaymentPage()   price dynamically incase it changes?

### DISCUSSION THREAD

**Discussion by adamstrad** [Glench/ExtPay#198] [Q&A] [answered]
I want to display my one-time purchase price dynamically in my html so that I can increase or decrease it after the app is published without having to go through the review process multiple times. Is there a simple object or function that exists that I can retrieve this from?

Example:
I have a "Lifetime access only ...$" button on my main menu, but maybe I want to put an offer on for a week.

I can simply change the price on my  ExtPay account, but it will not change the price displayed on my app

**Comment by Glench** [comment, marked as answer]
This isn't possible at the moment but stay tuned! For now what I do in my own extensions is to just not mention the price in my extension's client code.
